components/project-card/project-card.jsx

```diff
@@ -1,12 +1,95 @@
 import React from 'react';
 import ReactGA from 'react-ga';
 import { Link } from 'react-router';
+import Select from 'react-select';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import moment from 'moment';
 import { getBookmarks, saveBookmarks } from '../../js/bookmarks-manager';
+import Service from '../../js/service.js';
 import Utility from '../../js/utility.js';
 
+class ModerationPanel extends React.Component {
```

Comment:
Can we put these things in their own files? It's *much* easier to work with if components are in files named after the classes defined in them.